jbmc/regression/jbmc/simplify-classid-of-interface/test_no_propagation_vccs.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+Test.class
+--function Test.main --no-propagation --unwind 10 --show-vcc
```

Comment:
I think this should be `--no-simplify` instead of `--no-propagation`, because it's really the former that you are changing here.

Reply:
Done